pkg/paravirt/paravirt_protected_entity_type_manager.go

```diff
@@ -427,6 +428,7 @@ func (this *ParaVirtProtectedEntityTypeManager) createGuestPVC(ctx context.Conte
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: gcPVCNamespace,
 			Name:      gcPVCName,
+			Labels:    gcPVCLabels,
```

Comment:
What happened to svcPVC.Labels?  Do you see labels on Supervisor Cluster PVC?

With and Without your fix, can you do the following steps and save them in a doc?
1. Create a PVC in Guest Cluster with labels.
2. Check if labels are in both Guest and Supervisor Cluster PVC. Save "kubectl get pvc -o yaml".
3. Check vSphere UI Container Volumes and see if labels are there.  Provide a screen shot.
4. Backup the namespace including the PVC.
4. Delete the namespace from Guest Cluster.  Make sure PVC is deleted from Guest and Supervisor.
5. Check there are no Container Volumes in vSphere.  Take a screen shot.
6. Do a restore.  Save "kubectl get pvc -o yaml" from both Guest and Supervisor.
7. Take a screen shot of container volumes in vSphere.

Reply:
Updated documentation with records for each step.